docs/resources/cce_node.md

```diff
@@ -243,6 +322,52 @@ extend_param = {
   + `effect` - (Required, String, ForceNew) Available options are NoSchedule, PreferNoSchedule, and NoExecute.
     Changing this parameter will create a new resource.
 
+The `selectors` block supports:
+
+* `name` - (Required, String, ForceNew) Specifies the selector name, used as the index of `selector_names` in storage group.
+  The name of each selector must be unique. Changing this parameter will create a new resource.
+* `type` - (Required, String, ForceNew) Specifies the storage type. Currently, only **evs (EVS volumes)** and
+  **local (local volumes)** are supported. The local storage does not support disk selection.
```

Comment:
can we also add an example for local stroage?

Reply:
I checked the API and console again. `local` can only be used when creating a bare metal node with local disks.
But if I choose a bare metal flavor with local disks, for example: physical.d2.large, I can't config root_volume and data_volume for the node. This is conflict to our cce_node resource. So we can only support `evs` for now.

Reply:
if so, better to note that here.